.devcontainer/devcontainer.json

```diff
@@ -19,9 +19,9 @@
         "go.useLanguageServer": true,
         "go.goroot": "/opt/go",
         "go.testEnvVars": {
-            "GODEBUG": "cgocheck=2"
+            "GODEBUG": "cgocheck=2",
+            "CEPH_CONF": "/ceph_a/ceph.conf",
         },
-        "go.lintTool": "golangci-lint",
```

Comment:
Just curious: Why we have to remove this?

Reply:
I realised that shouldn't be set by the devcontainer settings, but by the user settings, because it's the choice of the developer. I prefer to use staticcheck or revive in vscode now, for instance, because it's faster.